docs/guides/applications/configuration-management/ansible/using-the-linode-ansible-collection-to-deploy-infrastructure/index.md

```diff
@@ -159,65 +196,59 @@ token: !vault |
-[linode.cloud.firewall](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/firewall.md)|Create and destroy Firewalls.
-[linode.cloud.firewall_info](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/firewall_info.md)|Gather info about an existing Firewall.
-[linode.cloud.firewall_device](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/firewall_device.md)|Manage Firewall Devices.
-[linode.cloud.instance](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/instance.md)|Create and destroy Linodes.
-[linode.cloud.instance_info](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/instance_info.md)|Gather info about an existing Linode instance.
-[linode.cloud.lke_cluster](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/lke_cluster.md)|Manage LKE clusters.
-[linode.cloud.lke_cluster_info](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/lke_cluster_info.md)|Gather info about an existing LKE cluster.
-[linode.cloud.nodebalancer](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/nodebalancer.md)|Create, destroy, and configure NodeBalancers.
-[linode.cloud.nodebalancer_info](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/nodebalancer_info.md)|Gather info about an existing NodeBalancer.
-[linode.cloud.nodebalancer_node](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/nodebalancer_node.md)|Manage NodeBalancer nodes.
-[linode.cloud.object_cluster_info](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/object_cluster_info.md)|Gather info about Object Storage clusters.
-[linode.cloud.object_keys](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/object_keys.md)|Create and destroy Object Storage keys.
-[linode.cloud.vlan_info](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/vlan_info.md)|Gather info about an existing Linode VLAN.
-[linode.cloud.volume](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/volume.md)|Create, destroy, and attach Linode volumes.
-[linode.cloud.volume_info](https://github.com/linode/ansible_linode/blob/v0.7.1/docs/modules/volume_info.md)|Gather info about an existing Linode volume.
-
-In the table above, each module can be selected to navigate your web browser to the public github page for the FQCN, which contain a list of all available configuration options for the resource the Module applies to.
-
-{{< note >}}
-The links in the above table lead to the github repository for version `v0.7.1` of the Linode Ansible Collection. For the latest version of the listed modules, it is recommended that users navigate to the [home page of the public Github repository](https://github.com/linode/ansible_linode) for a directory containing the latest version.
-{{< /note >}}
+[linode.cloud.domain](https://github.com/linode/ansible_linode/blob/main/docs/modules/domain.md)|Create and destroy domains.
```

Comment:
Made links point to `main` branch instead of tag `v0.7.1` so they are always up to date